systemtests/src/main/java/io/enmasse/systemtest/messagingclients/ExternalMessagingClient.java

```diff
@@ -142,6 +142,13 @@ public String getId() {
         }
     }
 
+    public boolean isSender() {
+        ClientType clientType = this.client.getClientType();
```

Comment:
Instead of checking this enum we should implement getCapability in clients, which returns SENDER/RECEIVER enum value and just check it.

Reply:
Agree, that was one of the things, that i wanted to discuss. There is also option to add some abstraction such as Sender/Receiver interface. But in this case I agree, get Capability could be best option.